server/index.js

```diff
@@ -21,14 +21,13 @@ app.get('/:coll/latest', (req, res) => {
     hostname: '127.0.0.1',
     port: '8080',
     path: '/' + req.params.coll + '/latest',
-    method: 'GET'
   }
-  http.request(options, cpuRes => {
```

Comment:
I also had a fix here to handle the situation where the http response is chunked (returned over multiple calls for serverRes.on('data') in this case).

app.get(['/plugins/*', '/api/*'], (req, res) => {
  var options = {
    hostname: '127.0.0.1',
    port: '8090',
    path: req.originalUrl,
    method: 'GET'
  }

  http.request(options, webRes => {
    webRes.setEncoding('utf8');

    let rawData = '';

    webRes.on('data', chunk => {
      rawData += chunk;
    });

    return webRes.on('end', ()=> {
        res.send(rawData);
    });
  }).end();

});


Basically adds data continuously and then on 'end', sends it.

Can you incorporate this change as well?

I can merge after.